tfx/proto/example_gen.proto

```diff
@@ -71,10 +76,10 @@ message Input {
   //     latest version for the latest span, and store the version number in
   //     metadata.
   //   - If VERSION is specified, but not SPAN, an error will be thrown.
+  //   - It could also be optionally specified with the width in the same way
+  //     as SPAN spec. e.g. 'data_span_{SPAN}_version_{VERSION:4}.data-*'.
```

Comment:
after 80 
+ For zero padding digits (e.g., `0012`), the width must be used.

Reply:
@hughmiao is this necessarily true though? I thought the OSS build of TFX has the ability to handle zero padding without specifying width.

Reply:
this is about the conversation we had last time, when in some mode, e.g., trigger example-gen with static range, we need to have the width information in order to reconstruct the path.

Reply:
gotcha i see, this is a precaution for that behavior, thanks!